app/src/test/java/org/greenstand/android/TreeTracker/messages/individualmessagelist/IndividualMessageListViewModelTest.kt

```diff
@@ -0,0 +1,74 @@
+package org.greenstand.android.TreeTracker.messages.individualmessagelist
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import io.mockk.coEvery
+import io.mockk.coVerify
+import io.mockk.mockk
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.runBlocking
+import org.greenstand.android.TreeTracker.MainCoroutineRule
+import org.greenstand.android.TreeTracker.messages.individualmeassagelist.IndividualMessageListViewModel
+import org.greenstand.android.TreeTracker.models.UserRepo
+import org.greenstand.android.TreeTracker.models.messages.MessagesRepo
+import org.greenstand.android.TreeTracker.utils.FakeFileGenerator
+import org.greenstand.android.TreeTracker.utils.getOrAwaitValueTest
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+@ExperimentalCoroutinesApi
+class IndividualMessageListViewModelTest {
+    @get:Rule
+    var instantTaskExecutorRule = InstantTaskExecutorRule()
+
+    @get:Rule
+    var mainCoroutineRule = MainCoroutineRule()
+    private val userId = 2L
+    private val userRepo = mockk<UserRepo>(relaxed = true)
+    private val messagesRepo = mockk<MessagesRepo>(relaxed = true)
+    private lateinit var individualMessageListViewModel: IndividualMessageListViewModel

```

Comment:
This is just preference, but I think naming whatever class you're testing as "testSubject" makes it very clear what's being tested. That allows you to look at any test and instantly make out what is being tested.